rcode-reader/src/main/java/eu/rssw/pct/elements/IndexComponentElement.java

```diff
@@ -0,0 +1,34 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class IndexComponentElement extends ClassElement {
+  protected int ascending;
+  protected int flags;
+  protected int position;
+
+  protected IndexComponentElement(byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "textAreaOffset". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172)